### PR TITLE
Mafia: Add host queue

### PR DESCRIPTION
**Pull request by ZestOfLife**

### CODE REVIEW

**Comment by whalemer** [comment]
might want to put a length limit on adds

File: chat-plugins/mafia.js [hunk 1]
```diff
@@ -2034,6 +2093,9 @@ const commands = {
 	mafiahelp: [
 		`Commands for the Mafia plugin:`,
 		`/mafia host [user] - Create a game of Mafia with [user] as the host. Requires + % @ * # & ~, voice can only host themselves.`,
+		`/mafia queue - Shows the upcoming users who are going to host.`,
+		`/mafia queue add, (user) - Adds the user to the hosting queue. Requires: + % @ # & ~`,
+		`/mafia queue remove, (user) - Removes the user from the hosting queue. Requires: + % @ # & ~`,
```
Comment (HoeenCoder) [inline chat-plugins/mafia.js]: Maybe combine this into one line?

File: chat-plugins/mafia.js [hunk 2]
```diff
@@ -1244,6 +1249,62 @@ const commands = {
 		},
 		hosthelp: [`/mafia host [user] - Create a game of Mafia with [user] as the host. Requires + % @ * # & ~, voice can only host themselves.`],
 
+		q: 'queue',
+		queue: function (target, room, user) {
+			if (!room.mafiaEnabled) return this.errorReply(`Mafia is disabled for this room.`);
+			if (room.id !== 'mafia') return this.errorReply(`This command can only be used in the Mafia room.`);
+			const args = target.split(',');
+			if (['forceadd', 'add', 'remove', 'del', 'delete'].includes(toId(args[0]))) {
+				if (!user.can('broadcast', null, room)) return this.errorReply(`/mafia ${toId(args[0])} - Access denied.`);
```
Comment (HoeenCoder) [inline chat-plugins/mafia.js]: `/mafia queue ${args[0]} - Access denied.`

File: chat-plugins/mafia.js [hunk 3]
```diff
@@ -1244,6 +1249,62 @@ const commands = {
 		},
 		hosthelp: [`/mafia host [user] - Create a game of Mafia with [user] as the host. Requires + % @ * # & ~, voice can only host themselves.`],
 
+		q: 'queue',
+		queue: function (target, room, user) {
+			if (!room.mafiaEnabled) return this.errorReply(`Mafia is disabled for this room.`);
+			if (room.id !== 'mafia') return this.errorReply(`This command can only be used in the Mafia room.`);
+			const args = target.split(',');
```
Comment (HoeenCoder) [inline chat-plugins/mafia.js]: Since your toIding the args alot, why not just do it here?
`const args = target.split(',').map(toId);`

**Comment by whalemer** [comment]
voices should be able to end games too

**Pull request merged**